bin/s2nd.c

```diff
@@ -680,7 +681,9 @@ int main(int argc, char *const *argv)
 
     GUARD_EXIT(s2n_config_add_dhparams(config, dhparams), "Error adding DH parameters");
 
-    GUARD_EXIT(s2n_config_set_cipher_preferences(config, cipher_prefs),"Error setting cipher prefs");
+    const struct s2n_security_policy *policy;
+    GUARD_EXIT(s2n_find_security_policy_from_version(cipher_prefs, &policy), "Error finding policy");
```

Comment:
We could be more explicit - "Error finding security policy"

Reply:
updated